test/worlds/particle_emitter.sdf

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="particle_emitters">
+
+    <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1.0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-scene-broadcaster-system"
+      name="ignition::gazebo::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-particle-emitter-system"
+      name="ignition::gazebo::systems::ParticleEmitter">
```

Comment:
I suspect you'll also need a `Scene` system for this to work when the implementation is complete, right?

Also, for testing purposes, you probably don't need `Physics` or `SceneBroadcaster`

Reply:
Tweaked in fb85188.